### ISSUE TITLE
Configured scripts without a run section are silently dropped

### ISSUE THREAD

**Issue by grayside** [phase2/rig#40]
If you configure a script such as:

```
scripts:
  ninja:
    description: Ninja vanish
```
or
```
scripts:
  ninja:
    run:
```
the script does not register with rig project.

There should be a verbose or warning level log explaining the incomplete script configuration.

Bonus points if we implement the check for this problem via a `rig project config` or `rig project validate` command.

**Comment by febbraro** [comment]
We validate every config file when we read it, we should just add that validation there, but have a way to determine what level of severity or scrutiny we place on validation (when running a project command we want lots of scrutiny)

**Comment by grayside** [comment]
Right, I'm suggesting we should have a command dedicated to that, and wondering if setting it up in that way requires changing our approach for further validation rules.

**Comment by febbraro** [comment]
What is the rationale for a separate command that you have to run, instead of just reporting these problems when the project specific configuration is in play?

**Comment by grayside** [comment]
Similar to the rationale of `docker-compose config -q` or `composer validate`, it means we can test the correctness/diagnose problems with the configuration before committing changes.

Tightening up the validation without regard for a separate command is the priority, but if possible we should let problems trickle up to the caller to determine what/how to report.

**Comment by febbraro** [comment]
Resolved in #43. If we want a `rig project validate` or `rig project config` then lets open another issues specifically for that.
